git.md

```diff
@@ -0,0 +1,50 @@
+# Working with Git
+
+These guidelines applied to **all** CocoaHeads Russia projects.
+
+## General
+
+We use [GitFlow](http://nvie.com/posts/a-successful-git-branching-model/) approach in our projects.
+
+![GitFlow](/resources/images/gitflow.png)
+
+## Branching
+
+**We use [Trello](https://trello.com) for task managements and [this hook](/resporces/scripts/prepare-commit-msg) for git integration with Trello**
+
+How it works?
+
+Each git brach name should match the following pattern `task/TRELLO_TASK_ID/task-description` **(e.g `task/D84aPyrz/create-models`)**. Hook looks for task id and push it to your commit message. You should write task id once in your branch name.
+
+**You need to place hook in `.git/hooks` folder**.
+
+## Commit
+
+- Use only English for commit comments
+- Comments must start from verb and describe main changes **(e.g. Fixed GitHub authorization bug, Added User model)**
```

Comment:
- Comments must start from verb and describe main changes
`e.g. Fixed GitHub authorization bug, Added User model`

Reply:
Lets use `code formatting` only for code. This is about commit comment so I would suggest to use text formatting here.

Reply:
You are right. What about quote? 
> Fixed GitHub authorization bug, Added User model